docs/eventing/samples/gcp-pubsub-source/trigger.yaml

```diff
@@ -28,5 +28,5 @@ spec:
     ref:
       apiVersion: serving.knative.dev/v1alpha1
       kind: Service
-      name: message-dumper
+      name: event-display
```

Comment:
Similar to the previous comment, the readme needs an updated verification section. I think s/message-dumper/event-display/g would work.